pandas/core/arrays/string_arrow.py

```diff
@@ -831,3 +831,30 @@ def _str_lower(self):
 
     def _str_upper(self):
         return type(self)(pc.utf8_upper(self._data))
+
+    def _str_strip(self, to_strip=None):
```

Comment:
ought to create an issue to remove the hasattr when we bump the required min pyarrow version